sdks/python/apache_beam/examples/snippets/transforms/aggregation/combinevalues_it_test.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import pytest
+import unittest
+
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+
+from . import combinevalues
+
+class CombineValuesIT(unittest.TestCase):
```

Comment:
I would suggest making this a validates runner test like:
https://github.com/apache/beam/blob/897fffdeda59cc2f121dd502e569770a897fc4ac/sdks/python/apache_beam/transforms/ptransform_test.py#L702

in ptransform_test.py